variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "ubuntu_ami" {
+  description = "The AMI to use for the server"
+  default = "ami-0c7217cdde317cfec"
+  
+}
+
+variable "free_tier_instance_type" {
```

Comment:
actually you should define tyoe of variables, because if your code will be used by someone, it would make additional difficulties to customize infra via this variables (especially while this variables too many)